src/plugins/winetricks.cpp

```diff
@@ -53,36 +53,60 @@ winetricks::winetricks(QString prefixName) : QWidget()
 
 QStringList winetricks::get_installed() {
 
-    QStringList installed_apps;
-
-    if (!check_script())
-        return installed_apps;
-
-    QString infile = QString(winetricks_bin);
-
-    QProcess process;
-
-    QStringList args; // = get_command("list-installed");
-    args.append("-c");
-    args.append(get_command("list-installed").join(" "));
-
-    QString exec = "/bin/sh";
-
-    process.start(exec, args);
-    process.waitForFinished(-1); // will wait forever until finished
-
-    QString stdout = process.readAllStandardOutput();
-    QString stderr = process.readAllStandardError();
-
-    QString pkg_name;
-    foreach(pkg_name, stdout.split("------------------------------------------------------").last().split("\n")){
-        if (!pkg_name.isEmpty()){
-            qDebug() << pkg_name;
-            installed_apps.append(pkg_name);
+    QStringList installed_verbs;
+    qstring_map name_val_settings;
+    QString winver = nullptr;
+    QHash<QString, QString> prefix_info = db_prefix.getByName(this->prefix_name);
+    QString logfile = QString("%1/winetricks.log").arg(CoreLib->getStrictEscapeString(prefix_info.value("path")));
+    QFile file(logfile);
+    if (!file.open(QIODevice::ReadOnly | QIODevice::Text)) {
+        return installed_verbs;
+    } else {
+        const QString winver_verbs[] = {
+            "nt40", "vista", "win2k", "win2k3", "win2k8", "win31", "win7",
```

Comment:
i don't really like the hardcoded values. maybe it is better to use QRegExp to match something like:

```
(nt[0-9]+)|(win[.*])|(vista)
```

other vice we will need to keep this in sync manually.